packages/enzyme-adapter-react-16.3/src/ReactSixteenThreeAdapter.js

```diff
@@ -483,6 +484,10 @@ class ReactSixteenThreeAdapter extends EnzymeAdapter {
     return typeOfNode(fragment) === Fragment;
   }
 
+  isForwardRef(forwardRef) {
```

Comment:
I agree that isCustomComponentElement can be an adapter competence because its implementation changes between React versions.

The problem is that I have to identify a forwardRef in a robust way and therefore I need to access the isForwardRef dependency of the react-is package available in the current dependencies of the adapter.

A forwardRef component is an object and not a function like other custom components, this is why isCustomComponentElement fails with them.

The current solution just add changes to 16 and 16.3 adapters, it's a similar solution of the current isFragment adapter method present only in the React 16 adapters.

The proposed solution to expand the current interface adapter to include the isCustomComponentElement method add changes to all the adapters but I agree can be a better solution.

Right now I can' t think of a cleaner or more robust solution than the previous ones, I'm sorry.

I wait for you to know your opinion about it.